__tests__/unit/gitlab/MergeRequestService.test.ts

```
@@ -0,0 +1,139 @@
+import { MergeRequestService } from '../../../app/api/gitlab/services/MergeRequestService';
+import { GitLabClient } from '../../../app/api/gitlab/client/GitLabClient';
+import { GitLabConfig } from '../../../app/api/gitlab/types';
```

ESLint

[error] 'GitLabConfig' is defined but never used.

(@typescript-eslint/no-unused-vars)